src/NzbDrone.Common/Disk/DriveInfoMount.cs

```diff
@@ -8,10 +9,11 @@ public class DriveInfoMount : IMount
         private readonly DriveInfo _driveInfo;
         private readonly DriveType _driveType;
 
-        public DriveInfoMount(DriveInfo driveInfo, DriveType driveType = DriveType.Unknown)
+        public DriveInfoMount(DriveInfo driveInfo, DriveType driveType = DriveType.Unknown, Dictionary<string,string> options = null)
```

Comment:
A dictionary or any enumerable type shouldn't be null, so you don't need to do a null check on it, so we should have a separate constructor or make it required.